index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <header class="header">
+        <div class="header-contain">
+            <div class="logo">My Logo</div>
```

Comment:
- [ ] If you notice most of the websites have logo as a link to `homepage`. 👍 .  This is because of their dependency on search engines, most users enter websites through an interior page and bypass the homepage entirely. Easy access to the homepage provides a new starting point for those who are on the wrong page or want to explore other parts of the site.
- Kindly ensure that you use `<a>` tag on the logo. 👍 👍